test/model/src/include/model/driver/kv_workload_runner.h

```diff
@@ -195,6 +195,23 @@ class kv_workload_runner {
         return table(op.table_id)->insert(transaction(op.txn_id), op.key, op.value);
     }
 
+    /*
+     * kv_workload_runner::do_operation --
+     *     Execute the given workload operation in the model.
+     */
+    int
+    do_operation(const operation::get &op)
+    {
+        try {
```

Comment:
question: isn't this try/catch block redundant?

Reply:
Yes actually, good catch. It's a hold-over from when I was using `get(...)` which throws exceptions, and I got so used to seeing it I didn't realize it was no longer necessary.